Cargo.toml

```diff
@@ -84,7 +84,7 @@ napi-build                = { version = "2.1.2" }
 napi-derive               = { version = "3.0.0-alpha.0", default-features = false, features = ["type-def"] }
 once_cell                 = "1.18.0"
 oxc                       = { version = "0.12.1", features = ["sourcemap"] }
-oxc_resolver              = { version = "1.2.0" }
+oxc_resolver              = { version = "1.6.4", features = ["package_json_raw_json_api"] }
```

Comment:
@hyf0 Which raw json value do you need? I don't want rolldown to use this, for saving a little bit of memory usage.